airflow/providers/google/cloud/transfers/azure_fileshare_to_gcs.py

```diff
@@ -63,13 +65,14 @@ class AzureFileShareToGCSOperator(BaseOperator):
         Service Account Token Creator IAM role to the directly preceding identity, with first
         account from the list granting this role to the originating account (templated).
 
-    Note that ``share_name``, ``directory_name``, ``prefix``, ``delimiter`` and ``dest_gcs`` are
+    Note that ``share_name``, ``directory_path``, ``prefix``, and ``dest_gcs`` are
     templated, so you can use variables in them if you wish.
     """
 
     template_fields: Sequence[str] = (
         "share_name",
         "directory_name",
+        "directory_path",
```

Comment:
Small bonus. If the deprecated parameter is templated then the new one should be as well.